Cannot compile C++ with clang++ when swiftly is installed

I get the following error compiling a C++ test file with swiftly installed:

```sh
❯ clang++ -std=c++20 test.cpp
test.cpp:1:10: fatal error: 'string' file not found
    1 | #include <string>
      |          ^~~~~~~~
1 error generated.
```

test.cpp
```cpp
#include <string>

int main() {

}
```

The C++ standard library header files should be available even with swiftly installed.

This version of clang++ cannot find the C++ standard library.
```sh
❯ which clang++
/Users/sbehnke/.swiftly/bin/clang++
```

This works as expected.
```sh
❯ /usr/bin/clang++ -std=c++20 test.cpp
```

I agree that it is the toolchain and not necessarily swiftly, but I'm not sure where else to report this. Is it possible to disable the swift 6.1 toolchain from swiftly when I'm not writing code in swift?

```sh
❯ $(swiftly use --print-location)/usr/bin/clang++ -std=c++20 test.cpp
test.cpp:1:10: fatal error: 'string' file not found
    1 | #include <string>
      |          ^~~~~~~~
1 error generated.
❯ swiftly use --print-location
/Users/sbehnke/Library/Developer/Toolchains/swift-6.1-RELEASE.xctoolchain
```

--- a/Sources/SwiftlyCore/Platform.swift
+++ b/Sources/SwiftlyCore/Platform.swift
@@ -200,6 +200,16 @@ extension Platform {
         for (key, value) in env {
             newEnv[key] = value
         }
+
+#if os(macOS)
+        // On macOS, we try to set SDKROOT if its empty for tools like clang++ that need it to
+        // find standard libraries that aren't in the toolchain, like libc++. Here we
+        // use xcrun to tell us what the default sdk root should be.
+        if newEnv["SDKROOT"] == nil {
+            newEnv["SDKROOT"] = (try? await self.runProgramOutput("/usr/bin/xcrun", "--show-sdk-path"))
+        }
+#endif
+
         try self.runProgram([command] + arguments, env: newEnv)
     }
 

--- a/Sources/SwiftlyCore/Platform.swift
+++ b/Sources/SwiftlyCore/Platform.swift
@@ -215,7 +215,7 @@ extension Platform {
         // find standard libraries that aren't in the toolchain, like libc++. Here we
         // use xcrun to tell us what the default sdk root should be.
         if newEnv["SDKROOT"] == nil {
-            newEnv["SDKROOT"] = (try? await self.runProgramOutput("/usr/bin/xcrun", "--show-sdk-path"))
+            newEnv["SDKROOT"] = (try? await self.runProgramOutput("/usr/bin/xcrun", "--show-sdk-path"))?.replacingOccurrences(of: "\n", with: "")
         }
 #endif
 

